react/index.js

```diff
@@ -81,7 +81,19 @@ class CategoryMenu extends Component {
       departmentsIds.includes(category.id)
     )
 
-    return (departmentsSelected.length && departmentsSelected) || categories
+    const departmentsWithIcons =
+      departmentsSelected.length &&
+      departmentsSelected.map(department => {
```

Comment:
Why doesn't `departmentsSelected` come with these properties in the first place?